src/components/Papers/PaperLine.jsx

```diff
@@ -33,7 +33,9 @@ const PaperLine = ({ paper, divider }) => {
       get(paper, `metadata.qualification[${paperDefinition.featureDate}]`),
     [paper, paperDefinition]
   )
-  const date = f(papersFeatureDate || paper.created_at, 'DD/MM/YYYY')
+  const paperLabel =
+    get(paper, 'metadata.qualification.pageName', null) || paper.name
```

Comment:
:thinking: what are the benefits to use lodash.get() here instead of `paper?.metadata?.qualification?.pageName` ?

Reply:
Also if using lodash, maybe `get(paper, 'metadata.qualification.pageName', paper.name)` would be better

Reply:
> 🤔 what are the benefits to use lodash.get() here instead of `paper?.metadata?.qualification?.pageName` ?

Plus une question d'habitude qu'autre chose, j'en profite pour enlever `lodash` de ce fichier 😁